config/abilities/master_admin.rb

```diff
@@ -48,6 +48,11 @@
     can :manage, :authentication_providers
     can :manage, :web_hooks
 
+    if user.account.provider_can_use?(:api_as_product)
```

Comment:
is this valid given master is not provider?

Reply:
As confirmed by @Martouta, provider means both master and tenants so yes it is.